distribution/test/itests/README.md

```diff
@@ -16,6 +16,16 @@ In short:
 * Use remote debugging to debug tests themselves or anything related to production code
 * Use local debugging to debug the code that sets up and configures the testing environment
 
+## Solr
+Solr runs in its own OS process separate PAX-EXAM and the application. Things to know include:
+
+* The default port for Solr is hardcoded to `9994`. 
```

Comment:
❓ Clarify how hardcoded? Is modifying the scripts sufficient or will a new distro need to be built.

Reply:
This value is in the itest pom and would need to be updated there, but not rebuilt. Added a note for developers on how to update.